src/traces/histogram2dcontour/defaults.js

```diff
@@ -30,5 +31,6 @@ module.exports = function supplyDefaults(traceIn, traceOut, defaultColor, layout
     if(autocontour) coerce('ncontours');
     else coerce('contours.size');
```

Comment:
It's subtle though. Sometimes these things *do* need re-coercing depending on the coupling between parameter defaults. But this one look like it's *maybe* redundant.